source/pdk/1.0/pdk_creating_modules.md

```diff
@@ -30,24 +30,29 @@ When you run the `pdk new module` command, the tool requests the following infor
 * The URL to a web site that offers full information about your module, if you have one.
 * The URL to the public bug tracker for your module, if you have one.
 
-After you generate a module, validate and test the module _before_ you add classes or write new code in it. This allows you to verify that the module files and directories were correctly created.
+After you create a module, validate and test the module _before_ you add classes or write new code in it. This allows you to verify that the module files and directories were correctly created.
 
-When you have validated the module, create classes by running the `pdk new class` command. The new class command also creates a test template file for the class. You can then write tests in this template to validate your class's behavior.
+When you have validated the module, create classes, defined types, and tasks by running `pdk` commands.
+
+The new class and defined type commands create a test template file for the class or defined type, so you can write tests in this template to validate your class's behavior. The new task command creates task and task metadata templates.
+
+When you create a task, PDK creates a task file in shell (`<TASK>.sh`), but you can write tasks in any language the target nodes will run. For more information about tasks, see the documentation about writing tasks.
```

Comment:
👍 could use a link to the correct docs.

Ah, I just found them below. Is this a general style thing in docs to have those links separate, instead of inline?

Reply:
It's considered a better practice, yes. Inline links can make users feel like they should drop what they're doing and go there now, and that's usually not needed. Also, this is the way that most links will be once everything is migrated to the new docs tooling, so we've been implementing it in newer docs as we go.

None of that is to say that inline links are ALWAYS bad; sometimes we will want them.

Reply:
👍